### PR TITLE
Update/category field

### PR DESCRIPTION
**Pull request by kmanaseryan**
Related issue:
https://github.com/GSA/datagov-ckan-multi/issues/526

Demo:
https://www.loom.com/share/009e1cfee67a41d7b776a8937902d7a3

### CODE REVIEW

File: metadata-app/src/components/AdditionalMetadata/index.js [hunk 1]
```diff
@@ -110,13 +112,9 @@ const AdditionalMetadata = (props) => {
       </div>
       <div className="grid-row margin-top-3">
         <div className="grid-col-12">
-          <WrappedField
-            label="Themes"
-            name="category"
-            type="string"
-            helptext="Main thematic category of the dataset.  If this dataset should be included in geoplatform.gov, please enter “geospatial” as the theme.  Start typing to add themes."
-            infoText={helpTexts.theme}
```
Comment (adborden) [inline metadata-app/src/components/AdditionalMetadata/index.js:118]: This help text is still useful, let's include the help text:
> Geospatial datasets are included in geoplatform.gov.

**Review by adborden — COMMENTED** [review]
Updating the copy to be consistent with our other radio button fields (Rights, Location, and Temporal).

**Comment by adborden** [comment]
@kmanaseryan can you check the contrast on the success message? I think the green font color is too light. Maybe we just want to use the USWDS success alert.

**Comment by kmanaseryan** [comment]
@adborden made the suggested changes, please check.

**Review by anuveyatsu — APPROVED** [review]
LGTM 👍

**Pull request merged**